code/modules/wiremod/shell/assembly.dm

```diff
@@ -23,8 +23,8 @@
 	), SHELL_CAPACITY_SMALL)
 	RegisterSignal(shell, COMSIG_SHELL_CIRCUIT_ATTACHED, PROC_REF(on_circuit_attached))
 	RegisterSignal(shell, COMSIG_SHELL_CIRCUIT_REMOVED, PROC_REF(on_circuit_removed))
-	RegisterSignals(src, list(COMSIG_ASSEMBLY_ATTACHED, COMSIG_ASSEMBLY_ADDED_TO_BUTTON), PROC_REF(on_attached))
-	RegisterSignals(src, list(COMSIG_ASSEMBLY_DETACHED, COMSIG_ASSEMBLY_REMOVED_FROM_BUTTON), PROC_REF(on_detached))
+	RegisterSignals(src, list(COMSIG_ASSEMBLY_ATTACHED, COMSIG_ASSEMBLY_ADDED_TO_OBJECT), PROC_REF(on_attached))
+	RegisterSignals(src, list(COMSIG_ASSEMBLY_DETACHED, COMSIG_ASSEMBLY_REMOVED_FROM_OBJECT), PROC_REF(on_detached))
 
 /obj/item/assembly/wiremod/proc/on_circuit_attached(_source, obj/item/integrated_circuit/circuit)
```

Comment:
I see where you got this _source from, yeah this is bad, likely got missed in review.
Could you quickly fix all of these _source param names to just `source`

Reply:
an _ at the front of a param name conventionally means the param isn't going to be used, but needs to be defined anyways, but sure, I can change this usage.

Reply:
ye but we don't follow that convention on tg